adserver/management/commands/archive_offers.py

```diff
@@ -135,7 +135,7 @@ def handle_archive_day(self, day):
 
     def copy_offer_dump(self, archive_filepath):
         """Copy offer CSV files to settings.BACKUPS_STORAGE."""
-        if not hasattr(settings, "BACKUPS_STORAGE"):
+        if "backups" not in settings.STORAGES:
```

Comment:
Not sure we're even using this file anymore? I guess we can keep it around for now.

Reply:
We can probably delete the whole management command. We're not using it.